mediUploadable.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  mediUploadable.swift
+//  watch-lib-tester-2 WatchKit Extension
+//
+//  Created by Nathaniel Jacobs on 11/21/19.
+//  Copyright © 2019 Medidata Solutions. All rights reserved.
+//
+
+import Foundation
+
+public protocol mediUploadable {
```

Comment:
Nitpick: protocol name should be capitlized

Reply:
Good catch, I'm confused from switching between so many languages